xen-bugtool

```diff
@@ -286,7 +282,6 @@ IPTABLES = 'iptables'
 ISCSIADM = 'iscsiadm'
 KPATCH = 'kpatch'
 LIST_DOMAINS = 'list_domains'
-LLDPTOOL = 'lldptool'
```

Comment:
Only as a side note, for reference (obsolete) as this PR is not needed:

This package is not a file from `fcoe-utils`:
```py
rpm -qf /usr/sbin/lldptool
lldpad-1.0.1-10.xs8.x86_64
```